### ISSUE TITLE
Android app - eldest called on user with existing eldest KID

### ISSUE THREAD

**Issue by notameadow** [keybase/client#6963]
This may be a duplicate of #6962 

Steps:
1. Fresh install/new user (account created via web, no other devices used)
2. Log it, get to the paper key screen
3. Lock the screen
4. Unlock 

Now you're back at the main screen and the paper key gone. I was expecting that since there is a switch to confirm that I wrote down the key all will be fine. 

5. Revoke the paper key
6. Revoke the device
7. Log in: "eldest called on user with existing eldest KID"

Removing app data does not help, neither does reinstalling.

**Comment by fbaube** [comment]
Yes, this is my experience too (#6962), and neither the app nor the CLI is clear about how to unbreak it.

**Comment by maxtaco** [comment]
Thanks for the report, we'll look right into it.

**Comment by maxtaco** [comment]
If either of you can send feedback through the app, that would be very useful too. Thanks!

**Comment by maxtaco** [comment]
BTW, have you guys reset your accounts recently?

**Comment by notameadow** [comment]
I cannot log it so I don't see an option to send feedback. I've created the account yesterday or so and the above were the only actions I took outside of the web interface (not much there as well).

**Comment by maxtaco** [comment]
Oh, I see. @notameadow it looks like you've revoked all of your devices and I probably need to reset your account.

@fbaube it looks like your device was provisioned successfully but somehow your provisioning didn't complete properly. Cc: @patrickxb on that issue.

**Comment by maxtaco** [comment]
@notameadow  I reset your account since all of your keys were revoked.  But there are still a few issues you must have hit to wind up there. 

Cc: @chrisnojima  --- what's the status of sending logs if people are all the way logged out?

**Comment by notameadow** [comment]
Reset as in delete and create again? So.. is it like I can destroy my account by revoking all keys/devices?

What about the problem with phone locking before I can find a pen? (I know it's a bit silly but still an issue).

**Comment by notameadow** [comment]
OK I can try to break it again if you like.

**Comment by maxtaco** [comment]
Yes, once you revoke all of your keys, your data is lost, as it should be.  The only way it wouldn't be lost is if we held onto a plaintext copy, which we obviously don't!

We need better seatbelts around revoking your last device, it's a known issue.

We're aware of the other problem about not having a pen.

But your experience should not suck as bad as it did if you wind up canceling the flow halfway through.  We're aware of that issue too, we'll hopefully be fixing that one first, since it's the most deadly..

**Comment by maxtaco** [comment]
it might be worth a try or you can wait for the next version -- up to you...

**Comment by notameadow** [comment]
I like living dangerously, it worked but those were nervous moments preventing the phone from locking and typing!
Thanks for that.

**Comment by fbaube** [comment]
I have no idea whether my account is reset or not. 
-Logging into Keybase.app (macos) tells me "has been revoked, please log in again".
-Trying to log in to Keybase on iPhone 4s, the Login button is obscured (screen layout on 4s is bad), so I press Enter and it returns to the create-or-login screen and says "Your device has been revoked".
-At macos CLI, "keybase login" works but then "keybase device list" tells me I can provision it from either paper (which I don't have; see original issue description) or from the phone (where Keybase is not letting me in). 
So, it's a riddle, wrapped in a mystery, inside an enigma. 
Is it reset ? Is it revoked ? Hmm, I can't create a NEW account with the same username...

**Comment by maxtaco** [comment]
Let me reset your account, is that OK?  haven't done so yet, been busy this morning

**Comment by fbaube** [comment]
BTW I did try earlier today to use macos to provision the phone (or was it the other way around) ?
macos CLI keybase produced a cool QR code but when i ointed the phone's Keybase at it, it crashed :-(

**Comment by fbaube** [comment]
Sure thing. Reset away !

**Comment by fbaube** [comment]
iPhone 4s app layout does need work. Cheers!

**Comment by fbaube** [comment]
OK, assuming it is reset, here's a stupid question: 
Should I start from macos CLI, macos app, or iPhone ?

**Comment by maxtaco** [comment]
We intend any path to work, but it might be smoother to start with the better tested desktop app. UI or CLI doesn't matter...

**Comment by fbaube** [comment]
Keybase.app got me back in OK.
Then tried to add the iPhone 4s.
Scanning QR code displayed on mac did not work - phone app showed the QR code in the camera "viewfinder" but it would not focus on the code.
Tried the text code instead - it seemed to accept the code, but then nothing happened.
Then trying to log in to the phone, but again, the Login button does not fit on an iPhone 4, and it will not switch to landscape mode, and hitting Enter bunced me back to the create-or-login screen.
So, the 4s has stymied me.

**Comment by Luki110998** [comment]
@maxtaco Could you reset my account, too? I have the same issue.

**Comment by maxtaco** [comment]
@Luki110998 I reset your account, you had revoked all of your keys.

**Comment by chrisnojima** [comment]
Hey @Luki110998 @fbaube @notameadow I'm curious why you're revoking all your devices. Is there some flow you're trying to achieve or are you just checking out how the software works?

**Comment by fbaube** [comment]
Basically just checking it out. I'm still stymied by the GUI layout on the iPhone 4s.

**Comment by gorgaar** [comment]
@maxtaco Could you please reset my account, too? I created it earlier to check it out and now I get the error message which led me here:

There was an error provisioning
Unkown error: eldest called on user with existing eldest KID

It's the same message on MacOS X and Android. Apparently I removed my device (Mac) before the Android device was set up and I didn't know that in that case I can't set up a new one from the web.

Thank you very much!

**Comment by hryamzik** [comment]
I've revoked my device in order to rename it and faced the same issue, could I possibly ask you to reset my account as well?

**Comment by maxtaco** [comment]
@gorgaar and @hryamzik, I reset your accounts

**Comment by bnschmdt** [comment]
Ahhh. "I've revoked my device in order to rename it and faced the same issue, could I possibly ask you to reset my account as well?" exact same issue here.

**Comment by maxtaco** [comment]
@bnschmdt  what is your keybase ID?

**Comment by bnschmdt** [comment]
"Benjamins"

Sent from my iPhone

On Jun 7, 2017, at 12:43, Maxwell Krohn <notifications@github.com<mailto:notifications@github.com>> wrote:


@bnschmdt<https://github.com/bnschmdt> what is your keybase ID?

**Comment by maxtaco** [comment]
ok, @bnschmdt, you are reset

**Comment by esunly** [comment]
Hi,

I cannot log in on any of my devices. I get this error: 

<img width="500" alt="screen shot 2017-06-08 at 12 33 19" src="https://user-images.githubusercontent.com/8968736/26924563-b30b244c-4c46-11e7-82b9-5da03d73b0f5.png">

Can anyone help?

**Comment by maxtaco** [comment]
ok @esunly all set to try again

**Comment by calinsth** [comment]
@maxtaco I hate to jump on this train, but I got antsy (revoked all devices) and now I have the same problem as above.  Can a homie get a reset? Keybase ID "c_lips".

**Comment by cd-mchase** [comment]
I need a reset as well.  Maybe I missed the spot in the ToS or the sign-up e-mail where if you remove your keys, you can't even login.  I was going to login to set up a different key because my web account wasn't recognizing that I had a cell phone key on my app and it kept asking me to scan the QR code.  I figured I'd remove the current key and just re-add a new one but now I can't even get into the account.  I can private message the Keybase ID.
